client/src/components/survey/PopUpContactAlert.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+const PopUpContactAlert = props => {
+  const { open, handleClose, styleButton, submitSurveyPage1, handleClickOpen } = props;
+
+  return (
+    <div>
+      <Button variant="outlined" color="secondary" className={styleButton} onClick={handleClickOpen}>
+        CONTINUE
+      </Button>
```

Comment:
I personally think this should be moved back into the Survey page, and you should just handle displaying the dialog with `setOpen` there, since this is just a dialog component, not a button + dialog component and we won't need to pass around as many props.